packages/data-contract/index.js

```diff
@@ -0,0 +1,64 @@
+const dotenv = require('dotenv')
+dotenv.config()
+
+
+const Dash = require('dash')
+
+function logInfo(...messages) {
+    console.log('\x1b[32m [INFO]', ...messages, '\x1b[0m ')
+}
+
+async function main() {
+    logInfo('Client Initialization')
+    
+    const schema = {
+        "dataContracts": {
+            "type": "object",
+            "properties": {
+                "identifier": {
+                    "type": "string",
+                    "minLength": 43,
+                    "maxLength": 44,
+                    "position": 0
+                },
+                "name": {
+                    "type": "string",
+                    "maxLength": 32,
+                    "minLength": 3,
+                    "position": 1.
+                }
+            },
+            "required": [
+                "shortName",
+                "contractId"
+            ],
+            "additionalProperties": false
+        }
+    };
+
+    let clientOpts = {
+        network: 'testnet',
+        wallet: {
+            mnemonic: process.env.MNEMONIC,
+            ...process.env.skipSynchronizationBeforeHeight && {
```

Comment:
I don't think you can specify environment variables in camelCase. it should be UPPER_CASE instead